src/main/java/com/google/api/codegen/config/DiscoApiModel.java

```diff
@@ -62,6 +62,12 @@ public boolean hasMultipleServices() {
     return interfaceModels;
   }
 
+  @Override
+  public Iterable<? extends TypeModel> getAdditionalTypes() {
+    // TODO: is this supported?
+    return ImmutableList.of();
```

Comment:
I think it's better to throw `UnsupportedOperationException`

Reply:
Yeah, I like that better too.

@landrito if you think there is a potential way to map this via discovery it's probably makes sense to file an issue as well. I'll defer that to you since I couldn't formulate a good description of the issue/work.